apps/test/unit/templates/VersionHistoryTest.jsx

```diff
@@ -210,11 +215,14 @@ describe('VersionHistory', () => {
       wrapper.find('.btn-danger').simulate('click');
 
       // Expect confirmation to show
-      expect(
+      assert(
         wrapper.containsMatchingElement(
           <div>
-            <p>Are you sure you want to clear all progress for this level?</p>
-            <button type="button" id="confirm-button">
+            <p>
+              Are you sure you want to restart this level? This will clear all
```

Comment:
NIT: why not use i18n here?

in general i am not a fan of this kind of test that just pastes in the html it's supposed to render, so i'd rather you find a more "logical" way to test this than clean this version up -- or simply leave it to be cleaned up as part of the more comprehensive dialog cleanup work that might be coming down the pipeline.

Reply:
Yeah, I dislike this too (which is why I wrote the new test differently).  Let me see if there's a quick fix; if not, I may just leave it.